types/types.go

```diff
@@ -9,6 +9,9 @@ const (
 
 	StateUp   = State("Up")
 	StateDown = State("Down")
+
+	AccessKey = "AWS_ACCESS_KEY_ID"
```

Comment:
`AWSAccessKey`, `AWSSecretKey`.

Keep in mind that we can support different protocols later, so the naming of variables shouldn't be too generic.